spec/system/placements/schools/placements/user_publishes_a_placement_after_preview_spec.rb

```diff
@@ -0,0 +1,221 @@
+require "rails_helper"
+
+RSpec.describe "User publishes a placement after preview", service: :placements, type: :system do
+  scenario do
+    given_that_placements_data_exists
```

Comment:
Would this be a simpler name for it? It's potentially a bit more readable for non-techie humans.

```suggestion
    given_that_placements_exist
```